nextjs-app/components/pages/program-rules/Schedule/ScheduleDetails.tsx

```diff
@@ -0,0 +1,7 @@
+import { FC, PropsWithChildren } from "react";
+
+const ScheduleDetails: FC<PropsWithChildren> = ({ children }) => {
+  return <div className="flex flex-col gap-6">{children}</div>;
+};
+
+export default ScheduleDetails;
```

Comment:
This is very simple and I don't think we need to extract to an independent component for it